netsetup/connman.py

```diff
@@ -96,3 +97,28 @@ def on_services_changed(currentServices, rmServices):
 
         wifi_tech.Scan(reply_handler=lambda: logging.info('Scan complete'),
                        error_handler=lambda err: logging.error(err))
+
+    def __get_wifi_service_path(self, name):
```

Comment:
Why not including `__get_wifi_service_path(self, name)` as a inner function of `__do_connect(self, path)`, and change it to `__do_connect(self, name)`?